model/host/host.go

```diff
@@ -567,27 +567,15 @@ func (h *Host) Upsert() (*mgo.ChangeInfo, error) {
 	)
 }
 
-func (h *Host) SetZoneAndStartTime() (*mgo.ChangeInfo, error) {
-	return UpsertOne(
-		bson.M{
-			IdKey: h.Id,
-		},
-		bson.M{
-			"$set": bson.M{
-				ZoneKey:      h.Zone,
-				StartTimeKey: h.StartTime,
-			},
-		},
-	)
-}
-
-func (h *Host) SetVolumeSize() (*mgo.ChangeInfo, error) {
+func (h *Host) CacheHostData() (*mgo.ChangeInfo, error) {
```

Comment:
do we ever do anything with the first result argument of this function? I think we shuldn't return it (and maybe error if nothing was userted or found (which shouldn't happen.